src/screens/Splash/OnBoardingSlides.tsx

```diff
@@ -172,10 +163,10 @@ const styles = StyleSheet.create({
     position: 'relative',
   },
   selectedDot: {
-    width: 25,
+    width: 26,
     height: 5,
     borderRadius: 5,
-    backgroundColor: '#E3BE96',
+    backgroundColor: '#89AEA7',
```

Comment:
We should avoid string literals and use the colorMode for better dark mode support